src/app/table-transactions/table-transactions.component.scss

```diff
@@ -0,0 +1,22 @@
+.table-transactions-component {
```

Comment:
`.table-transactions` will be enough for this name

Reply:
> ihor7016 2 days ago  Owner
> class name should match component name and file name